apps/docs/src/app/platform/component-docs/platform-forms/platform-date-picker/platform-date-picker-header/platform-date-picker-header.component.html

```diff
@@ -0,0 +1,21 @@
+<header>Date Picker</header>
+<description>
+    Platform DatePicker is an extension of the Date Picker component present in
+    <a href="https://sap.github.io/fundamental-ngx/#/core/datePicker">fundamental-ngx/core</a> to work with
+    <a href="https://sap.github.io/fundamental-ngx/#/platform/form">Platform Form</a>. <br />
+    Platform DatePicker keeps the same property and behaviour with core DatePicker.
+    <br />
+    <br />
+    <import module="PlatformDatePickerModule"></import>
+    <br />
+    <h3>Platform specific changes</h3>
+    <ul>
+        <li>
+            fundamental-ngx/core DatePicker has been Wrapped up to work with
+            <a href="https://sap.github.io/fundamental-ngx/#/platform/form">Platform Form</a>.
```

Comment:
Can we use routerlink. It helps to avoid whole page refresh

Reply:
used fdp-link now.